portal/src/main/webapp/index.jsp

```diff
@@ -17,20 +17,10 @@
     <script>
         window.frontendConfig = {
             configurationServiceUrl:"//" + '<%=baseUrl%>' +  "/config_service.jsp",
+            appVersion:'<%=GlobalProperties.getAppVersion()%>',
```

Comment:
this is already in frontendConfig.serverConfig.app_version

Reply:
so just discussed - this is necessary for cache buster, need to bust config service result itself